.travis.yml

```diff
@@ -30,14 +28,21 @@ cache:
   directories:
     - env/bin
     - env/lib/python2.7/site-packages
-    - node_modules
     - geckodriver
 install:
  - if [ "${TRAVIS_BRANCH}" = "master" -a "${TRAVIS_PULL_REQUEST}" = "false" ]; then rm -rf env; fi
  - touch requirements.txt package.json
  - make env -j2
 before_script:
  - echo "DATABASE_URL=dbname=gratipay" | tee -a tests/local.env local.env
+
+ # Turn off all cron threads.
+ - echo "UPDATE_CTA_EVERY=0" >> local.env
```

Comment:
All three envvar overrides on this commit are necessary:

- [`make bgrun`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/.travis.yml#L48) &rarr; [`$(honcho_run)`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L52) &rarr; [`local.env`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L12)
- [`make test`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/.travis.yml#L48) &rarr; [`$(pytest)`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L67) &rarr; [`$(test_env_files)`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L13) &rarr; [`tests/test.env`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L9)
- [`make doc`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/.travis.yml#L48) &rarr; [`$(doc_env_files)`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L116) &rarr; [`docs/doc.env`](https://github.com/gratipay/gratipay.com/blob/56526b9646938ffa0e4737eb9a04361b0d68f464/Makefile#L8)